src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -254,6 +288,18 @@ internal override bool IsValidInput(IServiceProvider serviceProvider, IConsole c
             var options = serviceProvider.GetRequiredService<IOptions<BuildScriptGeneratorOptions>>().Value;
             var logger = serviceProvider.GetRequiredService<ILogger<BuildCommand>>();
 
+            if (_languageWasSet)
+            {
+                logger.LogWarning("Deprecated option '--language' used");
+                console.WriteLine("Warning: the deprecated option '--language' was used.");
```

Comment:
Should we also recommend users to use the new name?

Reply:
I don't think we have that many users dependent on the old name. AFAIK this PR is solely for App Service